src/frontend/admin/src/utils/lang.ts

```diff
@@ -0,0 +1,28 @@
+import * as process from "process";
+import cookie from "@boiseitguru/cookie-cutter";
+import { DJANGO_LANGUAGE_COOKIE_NAME } from "@/utils/constants";
+import { LocalesEnum } from "@/types/i18n/LocalesEnum";
+
+export const DJANGO_FR_LANG = "fr";
```

Comment:
What about create a dedicated enum ( like you did for `LocalesEnum` ) for Django langs instead of redundant constants?